editor/assets/stylesheets/_mixins.scss

```diff
@@ -143,3 +143,60 @@ $float-margin: calc( 50% - #{ $visual-editor-max-width-padding / 2 } );
 	box-shadow: 0 0 0 1px $dark-gray-500;
 	color: $dark-gray-900;
 }
+
+/**
+ * Applies editor left position to the selector passed as argument
+ */
+@mixin editor-left( $selector ) {
```

Comment:
Is it possible to avoid the selector parameter and use `&`? that way, we could embed the `@include` call instead of passing the selector as a parameter. I don't know if it's possible personally but seems nicer if so.

Reply:
Hi @youknowriad, you mean use something like:
.components-notice-list {
@include editor-left;
}
Unfortunately, I don't think this is possible in our case. Even if we could use & all the styles included would be applied to a descendant of .components-notice-list